packages/components/src/Chip/Chip.types.d.tsx

```diff
@@ -1,11 +1,21 @@
 import React, { PropsWithChildren } from "react";
 
 export interface ChipProps extends PropsWithChildren {
+  /**
+   * If you need Chip to be part of a form element, this makes it not a button, but just a div so you can wrap it in a label.
+   */
+  readonly actAsFormElement?: boolean;
```

Comment:
Hmmm kinda misleading as it is still technically not a form element when used on its own. I get the need for it, but the prop name is odd.